docs/metrics/metrics-operators/quantize.md

```diff
@@ -13,12 +13,16 @@ You can specify:
 
 ## Syntax
 
-:::note
-The `quantize` operator *must* appear immediatdly after your query selector, before any other operators.
-:::
+There are two supported syntaxes for the `quantize` operator.
```

Comment:
just double-check: `quantize to INTERVAL` is not supported? (we might want also to explicitly mention it)

Reply:
`quantize to INTERVAL`  is supported. I guess that should be self-explanatory from first syntax:
```
quantize to INTERVAL [using ROLLUP] [drop last]
```
where `[]` denotes optional stuff. We also have an corresponding example.

Reply:
A follow up might be: why not to stuff both syntaxes in a single declaration. That would be possible with  some `[]` kung-fu, but I decided that such declaration would be too complex - it's much cleaner to declare two versions.

Reply:
ok, please ignore my comment, the proposal change looks ok ;)

Reply:
In reality, there are three versions:
1. `quantize to $interval`
2. `quantize using $add`
3. `quantize to $interval using $agg`

Saying there are two versions is even more misleading than `[]` kung-fu in my opinion. I'd write that we need to specify at least one of the two.

Reply:
Agree, my comment about not supporting `quantize to INTERVAL` was my first thought after I've read that we support 2 syntaxes

Reply:
I updated it to only one syntax definition and added a note about at least one of `to $interval` `using #rollup` being mandatory. Thanks for a good suggestion. 
Please check if it's clear for you now.